src/full/Agda/Utils/Update.hs

```diff
@@ -22,6 +22,8 @@ import Control.Monad.Fail (MonadFail)
 
 import Control.Monad.Identity
 import Control.Monad.Trans
+-- Control.Monad.Trans.Identity is already exported by Control.Monad.Identity
```

Comment:
But what is the harm?
I think we shouldn't remove support for older GHCs without need.

Reply:
This is not a removal but a comment that this can be removed at some future point when GHC 8.0.x is no longer supported.